storm-core/src/jvm/backtype/storm/windowing/TimeEvictionPolicy.java

```diff
@@ -21,21 +21,30 @@
  * Eviction policy that evicts events based on time duration.
  */
 public class TimeEvictionPolicy<T> implements EvictionPolicy<T> {
-    private final long duration;
+    private final int windowLength;
+    protected Long currentTs;
```

Comment:
What does currentTs represent? May I suggest renaming to something more representative?